core/domain/stats_services.py

```diff
@@ -1124,6 +1124,87 @@ def get_learner_answer_details(entity_type, state_reference):
     return None
 
 
+def get_learner_answer_info_for_exploration(exp_id):
+    """Returns the learner answer info in dict format.
+
+    Args:
+        exp_id: str. The ID of the exploration.
+
+    Returns:
+        list. Consists of dicts
+            with state name as keys, and a list of learner answer info dicts as
+            values. The learner answer info dicts contains following keys:
+                learner_answer_info_id: str. The id of the LearnerAnswerInfo
+                    object.
+                answer: json-like. The answer which is submitted by the
```

Comment:
json-like --> * (json-like).

(* means anything, and the parenthetical clarifies that it is of the json variety as opposed to objects etc.)

Reply:
Made services to return domain object only

Reply:
Ditto, are you addressing the wrong comment here?

Reply:
Should I also add this ```(* means anything, and the parenthetical clarifies that it is of the json variety as opposed to objects etc.) in the docstrings```

Reply:
No need, that was an explanation for you.